tests/test_data_adapter.py

```diff
@@ -267,6 +284,22 @@ def test_geodataset(geoda, geodf, ts, tmpdir):
     assert da3.vector.crs.to_epsg() == 4326
     with pytest.raises(FileNotFoundError, match="No such file"):
         data_catalog.get_geodataset("no_file.geojson")
+    da3 = data_catalog.get_geodataset(
+        "test.nc",
+        # only really care that the bbox doesn't intersect with anythign
+        bbox=[12.5, 12.6, 12.7, 12.8],
+        handle_nodata=NoDataStrategy.IGNORE,
+    )
+
+    with pytest.raises(NoDataException):
+        da3 = data_catalog.get_geodataset(
+            "test.nc",
+            # only really care that the bbox doesn't intersect with anythign
+            bbox=[12.5, 12.6, 12.7, 12.8],
+            handle_nodata=NoDataStrategy.RAISE,
+        )
+
+    assert len(da3) == 0
```

Comment:
Nice to see some tests added to this as well! Just for this line, it makes more sense to move it up (to 293), to keep it closer to code where the test belongs to.